js/global.d.ts

```diff
@@ -1,2 +1,6 @@
 declare module "*.less";
 declare module "*.png";
+declare module "*.svg" {
+  const content: any;
+  export default content;
```

Comment:
Out of curiosity, what's the significance of these `content` declarations?

Reply:
I modeled it after how it was handled in GeoCode.  Let me see if it behaves the same without them.

Reply:
Works fine without it.  Maybe originated from somewhere like here?
https://stackoverflow.com/questions/54121536/typescript-module-svg-has-no-exported-member-reactcomponent
I'm going to remove it since it seems unnecessary.